feature1/one-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was created in feature1 folder
+
+One more change in feature1 branch
```

Comment:
it's better to spell out name of branch into square brackets [feature1]

Reply:
Thanks for reply